interfaces/matlab/toolbox/@ThermoPhase/charges.m

```diff
@@ -0,0 +1,12 @@
+function ch = charges(tp)
+% CHARGES  Get the array of species charges 
+% x = speciesCharges(tp)
+%
+% :param tp:
+%     Instance of class :mat:func:`ThermoPhase` (or another
+%     object that derives from ThermoPhase)
+% :return:
+%     Vector of species charges. Units: elem. charge
+%
+
+ch = phase_get(tp.tp_id, 23);
```

Comment:
You need a newline at the end of this file

Reply:
Just an extra line below line 12? I added that and pushed, but I'm not seeing it on my github repo.

Reply:
At least in my view (I think I'm using the new interface) it shows as a red line with a circle in it when there's no new line in the Files Changed view on the PR. The problem seems to be fixed by 32ec0f5 so it's all good. See here for a brief reasoning on why files should end with a blank line: https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file